support/src/lib.rs

```diff
@@ -20,6 +20,9 @@
 pub mod deposit;
 pub use deposit::{free_deposit, reserve_deposit};
 
+#[cfg(any(feature = "try-runtime", test))]
+pub mod test;
```

Comment:
Not sure this is the best nemaing. This is a testing utility, right? Maybe `test_utils` or `test_utilities` would be better? We usually use `test` for actual tests.

Reply:
Also the function name itself is not very indicative, and a comment would not hurt about its expected usage.